packages/api-client-core/src/operationBuilders.ts

```diff
@@ -96,6 +98,31 @@ export const findManyOperation = (
   ]);
 };
 
+export const findFirstOperation = (
```

Comment:
I double-checked, but the disadvantage that would would be that you'd be getting back more data, so there's a bandwidth cost (at least).

But I might be able to compose it using `findMany`, just passing down that `first` variable.

The other thing is, I stripped these from the return fields:
`pageInfo: ["hasNextPage", "hasPreviousPage", "startCursor", "endCursor"]`
`edges: ["cursor"]`
...leaving just the node itself.

I think removing those makes sense, since this should have no pagination, but I _could_ add a boolean input param to `findManyOperation`, and use that to control the query, but at that point, it's not really just a `findMany` operation.

What if I followed a pattern I used elsewhere here, and had a 3rd, query construction function, with just the shared stuff? That avoids repetition, and enables composition, but without getting into confusing semantics.

Reply:
Oh, that's a good point Gedge. I'll just follow the same composition pattern as `findOneByFieldOperation` then, to be consistent. Thanks!